tck-dist/src/main/starter/se-pom.xml

```diff
@@ -99,28 +102,29 @@
    <artifactId>sigtest-maven-plugin</artifactId>
    <version>${sigtest.version}</version>
   </dependency>
-  <!-- APIs referenced by TCK that do not require implementations for standalone 
-   tests -->
+  <!-- APIs referenced by TCK that do not require implementations for standalone tests -->
   <dependency>
    <groupId>org.jboss.shrinkwrap</groupId>
    <artifactId>shrinkwrap-api</artifactId>
-   <version>1.2.6</version>
+   <version>${shrinkwrap.version}</version>
   </dependency>
   <dependency>
    <groupId>org.jboss.arquillian.junit5</groupId>
    <artifactId>arquillian-junit5-core</artifactId>
-   <version>1.7.0.Alpha13</version>
+   <version>${arquillian.version}</version>
   </dependency>
   <dependency>
    <groupId>jakarta.servlet</groupId>
    <artifactId>jakarta.servlet-api</artifactId>
-   <version>6.0.0</version>
+   <version>${jakarta.servlet.version}</version>
   </dependency>
+ <!-- APIs referenced by TCK that do require implementations for standalone tests -->
   <dependency>
    <groupId>jakarta.enterprise</groupId>
    <artifactId>jakarta.enterprise.cdi-api</artifactId>
-   <version>4.0.1</version>
+   <version>${jakarta.enterprise.cdi.version}</version>
   </dependency>
+  <!-- TODO add a CDI implementation that is supported by your Jakarta Data implementation -->
```

Comment:
If having and supporting `@Inject` is what is wanted and expected, then having a dependency on `jakarta.inject:jakarta.inject-api` and saying _DI framework implementation_ probably expresses that better. CDI itself is just a DI framework, so it satisfies that...